src/integrations/feed/types.ts

```diff
@@ -0,0 +1,66 @@
+import { Context } from '../../Context';
+
+export type FeedResponse = [postId: string, metadata: string | undefined][];
+
+export type FeedProvider = {
```

Comment:
With these options, do we actually want to make these known to API?
Is it not something feed service handles.

Mainly raising it to have the API be as dumb as possible and simple request a feed with some basic props.

Reply:
Actually, I'm gonna use it in the next PR partially. We actually want to give the API the flexibility to play around and experiment with some stuff based on user's state. But we can add it as necessary if you prefer

Reply:
Hmm might just make things messy from API side.
From that perspective I'd hoped most logic stayed on feed service, but open to discussion

Reply:
Ok, let's talk about it when we get the chance. And we can have stricter rules in case someone changes it